common/install_gcc-9.2.sh

```diff
@@ -1,29 +1,33 @@
 #!/bin/bash
-set -x
+set -ex
 
 MODULE_FILES_DIRECTORY=$1
 
 # Install gcc 9.2
 wget ftp://gcc.gnu.org/pub/gcc/infrastructure/gmp-6.1.0.tar.bz2
+../../../common/verify_checksum.sh gmp-6.1.0.tar.bz2 "498449a994efeba527885c10405993427995d3f86b8768d8cdf8d9dd7c6b73e8"
 tar -xvf gmp-6.1.0.tar.bz2
 cd ./gmp-6.1.0
 ./configure && make -j$(nproc) &&  make install
 cd ..
 
 wget ftp://gcc.gnu.org/pub/gcc/infrastructure/mpfr-3.1.4.tar.bz2
+../../../common/verify_checksum.sh mpfr-3.1.4.tar.bz2 "d3103a80cdad2407ed581f3618c4bed04e0c92d1cf771a65ead662cc397f7775"
 tar -xvf mpfr-3.1.4.tar.bz2
 cd mpfr-3.1.4
 ./configure && make -j$(nproc) &&  make install
 cd ..
 
 wget ftp://gcc.gnu.org/pub/gcc/infrastructure/mpc-1.0.3.tar.gz
+../../../common/verify_checksum.sh mpc-1.0.3.tar.gz "617decc6ea09889fb08ede330917a00b16809b8db88c29c31bfbb49cbf88ecc3"
 tar -xvf mpc-1.0.3.tar.gz
 cd mpc-1.0.3
 ./configure && make -j$(nproc) &&  make install
 cd ..
 
 # install gcc 9.2
 wget https://ftp.gnu.org/gnu/gcc/gcc-9.2.0/gcc-9.2.0.tar.gz
+../../../common/verify_checksum.sh gcc-9.2.0.tar.gz "a931a750d6feadacbeecb321d73925cd5ebb6dfa7eff0802984af3aef63759f4"
```

Comment:
Suggestion: Does it make sense to have a script in top folder that sets the "root_path" ($topdir) and source essential utilities (like download and check checksum)? This way, each script file doesn't have to do this "../../../" and use ${topdir}/<path> instead.